css/css-variables/variable-cssText.html

```diff
@@ -28,15 +28,15 @@
     "use strict";
 
     var testcases = [
-        { element: "target1",   expectedCssText: "--var:  var1;" },
+        { element: "target1",   expectedCssText: "--var: var1;" },
         { element: "target2",   expectedCssText: "margin: var(--prop);" },
         { element: "target3",   expectedCssText: "background: var(--prop);" },
         { element: "target4",   expectedCssText: "margin: var(--prop)  !important;" },
         { element: "target5",   expectedCssText: "background: var(--prop)  !important;" },
         { element: "target6",   expectedCssText: "background: green;" },
         { element: "target7",   expectedCssText: "background: var(--prop);" },
-        { element: "target8",   expectedCssText: "color:  var(--prop);" },
-        { element: "target9",   expectedCssText: "margin-top: 10px; margin-right: ; margin-bottom: ; margin-left: ;" },
+        { element: "target8",   expectedCssText: "color: var(--prop);" },
+        { element: "target9",   expectedCssText: "margin-right: ; margin-bottom: ; margin-left: ; margin-top: 10px;" },
```

Comment:
In Firefox this fails due to no whitespace between the property and the semicolon. I think this is a bug in Firefox per step 3 of https://drafts.csswg.org/cssom/#serialize-a-css-declaration, but just wanted to make sure I wasn't missing something.

Reply:
I don't think this serialization is any useful. We may want to update the spec to something so that when a longhand is expanded from a shorthand which references variables, the shorthand is always serialized, and any longhand doesn't share the same shorthand expression are not considered serialized by this shorthand.

Reply:
But that's a separate issue, and doesn't affect this change, I think.